RecoTracker/SpecialSeedGenerators/src/OutsideInMuonSeeder.cc

```diff
@@ -178,10 +184,19 @@ OutsideInMuonSeeder::produce(edm::Event & iEvent, const edm::EventSetup & iSetup
             }
         }
         if (tk.eta() > minEtaForTEC_) {
-            int iLayer = 9, found = 0;
-            std::vector< ForwardDetLayer const* > const & tec = measurementTracker->geometricSearchTracker()->posTecLayers();
-            for (auto it = tec.rbegin(), ed = tec.rend(); it != ed; ++it, --iLayer) {
-                if (debug_) std::cout << "\n ==== Trying TEC " << +iLayer << " ====" << std::endl;
+            auto forwLayers = measurementTracker->geometricSearchTracker()->posTecLayers();
```

Comment:
Yes, I discussed that separately with @ebrondol once she pointed out that the variable could take different values. I think `const auto&` works for this usage.